run_integration_tests.sh

```diff
@@ -5,20 +5,22 @@ set -e
 rootdir=`dirname $0`
 pushd $rootdir
 
-# get feature files from algorand-sdk-testing repo
-sudo rm -rf temp
-git clone --single-branch --branch templates https://github.com/algorand/algorand-sdk-testing.git temp
+# Reset test harness
+rm -rf test-harness
+git clone --single-branch --branch will/indexer https://github.com/algorand/algorand-sdk-testing.git test-harness
```

Comment:
this will need to become branch `develop` if the corresponding sdk-testing PR gets merged yeah?